edm4hep.yaml

```diff
@@ -227,3 +265,118 @@ datatypes :
       - edm4hep::Cluster        clusters     //clusters that have been combined to this cluster.
       - edm4hep::CalorimeterHit hits         //hits that have been combined to this cluster.
       - edm4hep::ParticleID     particleIDs  //particle IDs (sorted by their likelihood) 
+
+  #-------------  TrackerHit
+  edm4hep::TrackerHit:
+    Description : "Tracker hit"
+    Author : "F.Gaede, DESY"
+    Members :
+      - unsigned long long cellID      //ID of the sensor that created this hit
+      - int type                       //type of raw data hit, either one of edm4hep::TPCHIT, edm4hep::SIMTRACKERHIT - see collection parameters "TrackerHitTypeNames" and "TrackerHitTypeValues". 
+      - int quality                    //quality bit flag of the hit.    
+      - float time                     //time of the hit.
+      - float eDep                     //energy deposited on the hit [GeV]. 
+      - float eDepError                //error measured on EDep [GeV].
+      - float edx                      //dE/dx of the hit in [GeV]. 
```

Comment:
```suggestion
      - float edx                      //dE/dx of the hit in [GeV/mm]. 
```
?

Reply:
@gaede I believe Andre is correct, right? The comments in LCIO are sometimes [GeV](https://github.com/iLCSoft/LCIO/blob/568e2075843ea8c6ef3ee852a591633dedc85a17/src/cpp/src/UTIL/Operators.cc#L1780) and sometimes [GeV/mm](https://github.com/iLCSoft/LCIO/blob/27c67df622c934f3258a38e167b4b358781f306a/src/cpp/include/IMPL/TrackerHitZCylinderImpl.h#L66)